pywincffi/kernel32/handle.py

```diff
@@ -120,3 +120,50 @@ def WaitForSingleObject(hHandle, dwMilliseconds):
     error_check("WaitForSingleObject")
 
     return result
+
+
+def GetHandleInformation(hObject):
+    """
+    Returns properties of an object handle.
+
+    .. seealso::
+
+        https://msdn.microsoft.com/en-us/library/windows/desktop/ms724329(v=vs.85).aspx
```

Comment:
I've seen it in a couple of training sessions and I played around with the url structure some too.  Microsoft's robots.txt and sitemap tends to reference the direct urls, which is part of the reason Google's bot sees the full url.  I was a bit aprehensive about using the shorter urls but after investigating it some, they're not redirects:

```
> curl -vL https://msdn.microsoft.com/en-us/library/ms724329
...
> GET /en-us/library/ms724329 HTTP/1.1
> Host: msdn.microsoft.com
> User-Agent: curl/7.43.0
> Accept: */*
>
< HTTP/1.1 200 OK
...
```

Fun fact, this works too and might be something I update to later (I assume Microsoft's application will select the correct language): https://msdn.microsoft.com/library/ms724329